src/r/recordFactories/ProjectFactory.ts

```diff
@@ -1160,7 +1160,7 @@ export class ProjectUtils {
       this.accentColorsDict[accentColor.toLowerCase()] :
```

Comment:
```javascript
const idsForAccentColor: number[] = accentColor ?
      this.accentColorsDict[accentColor.toLowerCase()] || []:
      flatten(Object.values(this.accentColorsDict));

```

Reply:
If you add the condition and it's falsy, the control will flow to the rest of the code where it will start to filter out entries based on search string and return all rules eventually. 

What we want is that if there are no entries for a color, show blank rules, not all rules